bundles/configuration.rst

```diff
@@ -92,6 +92,13 @@ bundle configuration would look like:
             'client_secret' => 'your_secret',
         ));
 
+
+.. tip::
+
+    The root node of your bundle configuration must match the name of your 
+    bundle without the 'Bundle' suffix in camel case. For example 
+    ``AcmeSocialBundle`` becomes ``acme_social``.
+
```

Comment:
Indeed. The current description is not good enough! I propose we make this even *more* obvious. Instead of a note down here, let's move this into the text below the `Using the Bundle Extension` headline. Something like this:

> Imagine you are creating a new bundle -`AcmeSocialBundle` - which provides...
> integration with Twitter, etc. To make your bundle easy to use, you want to
> allow users to configure it with some configuration that looks like this:

... the code block

The root key - ``acme_social`` is automatically determined from your bundle name.

But, there's another problem: we never actually show the extension class being created! This lives in `/bundles/extension`. We should really move that content into this file.

This just got a bit bigger @mickadoo. Are you up for trying it? :)

Reply:
Ryan, I've made most of the changes you asked here while merging ... but I didn't include the extension example because it's too complex. Thanks!